frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx

```diff
@@ -60,16 +69,17 @@ const TextColumnField: React.FC<TextColumnFieldProps> = ({
                     </FlexItem>
                     {!isReadOnly && (
                       <FlexItem>
-                        <MinusCircleIcon
-                          aria-hidden="true"
-                          style={{ color: rowValues.length === 1 ? disabledColor.value : null }}
+                        <Button
+                          variant={ButtonVariant.plain}
+                          type={ButtonType.button}
+                          isInline
+                          isDisabled={disableDeleteRow}
                           onClick={() => {
-                            if (rowValues.length === 1) {
-                              return;
-                            }
                             arrayHelpers.remove(idx);
                           }}
-                        />
+                        >
+                          <MinusCircleIcon aria-hidden="true" />
```

Comment:
Not sure why you need aria-hidden.
```suggestion
                          <MinusCircleIcon />
```

Reply:
removed it.